### ISSUE TITLE
Ask for advice on the details of algorithm reproduction

### ISSUE THREAD

**Issue by Kakako2060** [dunzeng/FedAWARE#4]
`

[fedAWARE.zip](https://github.com/user-attachments/files/19332619/fedAWARE.zip)

`

![Image](https://github.com/user-attachments/assets/e9859f45-880c-4f46-b01e-3f8a8f813524)

Hello author: I am a beginner to federated learning, and I am currently reproducing your fedAWARE algorithm, but I have encountered the problem of poor training effect in the specific implementation process, and would sincerely appreciate your guidance on potential implementation errors. 
I've uploaded the attached files for the current implementation, along with the training test graph on the CIFAR10.
 Thank you so much for taking the time to review this issue! Looking forward to your guidance.

**Comment by dunzeng** [comment]
<img width="969" alt="Image" src="https://github.com/user-attachments/assets/e343b572-7314-4b3f-8d8b-42fa56eb389f" />

Please remove lines 119-120, which correct your codes to conduct server-side gradient descent with FedAWARE estimates.

**Comment by Kakako2060** [comment]
Ok, thank you very much for your reply, I just tried it and found that the training had no effect. This kind of training has not changed directly, I don't understand it very well, is it convenient to chat privately, or can you please provide some more detailed guidance if you have time Thank you very much.

![Image](https://github.com/user-attachments/assets/5060c04f-1cf0-4b8f-9396-63cef111c3ed)

> <img alt="Image" width="969" src="https://private-user-images.githubusercontent.com/32679207/424296885-e343b572-7314-4b3f-8d8b-42fa56eb389f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzNjA2NDEsIm5iZiI6MTc0MjM2MDM0MSwicGF0aCI6Ii8zMjY3OTIwNy80MjQyOTY4ODUtZTM0M2I1NzItNzMxNC00YjNmLThkOGItNDJmYTU2ZWIzODlmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDA0NTkwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwMTgxZDE5YjEwYjBlZTVkYzY0NmNkNDZjNTc5NmI5NjcwMGE1MTNmOGFjYmZhNDljYzUwYTY2NGY5MjJlYWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-y9TubwsxgSQvIEQhYpIHXBUZxEny0nvF8pCaasjlN0">
> Please remove lines 119-120, which correct your codes to conduct server-side gradient descent with FedAWARE estimates.

**Comment by dunzeng** [comment]
I'm not familiar with the framework you are using. Providing more details about the experiment settings and training hyperparameters can be helpful. 

I'm also ok to chat privately if you leave your contact.

**Comment by Kakako2060** [comment]
> I'm not familiar with the framework you are using. Providing more details about the experiment settings and training hyperparameters can be helpful.
> 
> I'm also ok to chat privately if you leave your contact.

ZHY_2060KKZ
